acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/Signer.java

```diff
@@ -57,14 +55,17 @@
       "/api/v1/eth2/sign/{identifier}"; // using bls keys
   public static final String ETH1_PUBLIC_KEYS = "/api/v1/eth1/publicKeys"; // secp keys
   public static final String ETH2_PUBLIC_KEYS = "/api/v1/eth2/publicKeys"; // bls keys
+  public static final ObjectMapper ETH_2_INTERFACE_OBJECT_MAPPER =
+      new ObjectMapper()
+          .registerModule(new SigningJsonRpcModule())
```

Comment:
probably wrong module - i.e there should be a HttpApi Module - or this module needs to be renamed to be generic?

Reply:
The module isn't generic though, it's specific the signing APIs. The filecoin module uses some of the same types e.g. bytes and represents them differently.

Reply:
But states "SIgningJsonRpcModule()" <-- i.e. I had assumed that meant Filecoin?